package.json

```diff
@@ -22,7 +22,7 @@
     "prettier:check": "prettier --check '**/*.{jsx,js,ts,tsx,md,mdx}'",
     "prebuild": "lerna run prebuild",
     "lerna:version": "lerna version --no-private --conventional-commits --create-release github --yes",
-    "lerna:publish": "lerna publish from-git --yes",
+    "lerna:publish": "npx lerna exec -- npm publish",
```

Comment:
using `npx` instead of yarn for this step as yarn seems to not respect `.npmrc` for some reason.

Reply:
cool literally commented on this the same time you did haha https://github.com/contentful/rich-text/pull/424#discussion_r1037067234

Reply:
Umm... thinking about it again, I think this may fail in CI because we run `yarn lerna:publish` 🤔

Reply:
Gonna add this command in the config.yaml directly. At least there I can add a comment explaining why